DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -248,13 +279,30 @@ public static bool IsInRole(string role)
             return IsInRoles(UserController.Instance.GetCurrentUserInfo(), PortalController.Instance.GetCurrentPortalSettings(), role);
         }
 
+        /// <summary>
+        /// Determines whether the current user belongs to the specified roles.
+        /// </summary>
+        /// <param name="roles">The semicolon separated list of roles.</param>
+        /// <returns>
+        ///   <c>true</c> if user belongs to the specified roles; otherwise, <c>false</c>.
+        /// </returns>
         public static bool IsInRoles(string roles)
         {
             UserInfo objUserInfo = UserController.Instance.GetCurrentUserInfo();
             PortalSettings settings = PortalController.Instance.GetCurrentPortalSettings();
             return IsInRoles(objUserInfo, settings, roles);
         }
 
+        /// <summary>
+        /// Determines whether the provided user belongs to the specified roles.
+        /// </summary>
+        /// <param name="objUserInfo">The user information.</param>
+        /// <param name="settings">The settings.</param>
+        /// <param name="roles">The semicolon separated list of roles.</param>
+        /// <returns>
+        ///   <c>true</c> if the provided user belongs to the specific roles; otherwise, <c>false</c>.
+
+        /// </returns>

```

Comment:
```suggestion
        /// <summary>
        /// Determines whether the provided user belongs to the specified roles.
        /// </summary>
        /// <param name="objUserInfo">The user information.</param>
        /// <param name="settings">The settings.</param>
        /// <param name="roles">The semicolon separated list of roles.</param>
        /// <returns>
        ///   <c>true</c> if the provided user belongs to the specific roles; otherwise, <c>false</c>.
        /// </returns>
```